include/Platform/Universal Windows/UWP/RTHelpers.h

```diff
@@ -51,8 +51,8 @@
 #include <UWP/interopBase.h>
 
 #if __has_feature(objc_arc)
-#define AUTORELEASE(x)
-#define RETAIN(x)
+#define AUTORELEASE(x) x
```

Comment:
There's an extra x floating around on the two lines. Don't see that in develop or packaging.